### DISCUSSION TITLE
请问如何实现分页查询

### DISCUSSION THREAD

**Discussion by abadfox233** [illacloud/illa-builder#622] [Q&A] [answered]
数据库里面有几万条数据, 全量查询会出现的 write: broken pipe  的错误

![image](https://user-images.githubusercontent.com/32118362/205307873-f43c7779-fd8c-4663-9e0f-ff9b786678d0.png)

 但是界面上没有摸索出能够实现分页查询的实现方法,
> * 尝试通过环境变量分页, 失败
> * 尝试组合其他控件分页, 方案可行, 但是太丑陋了

**Comment by wenzhuo2022** [comment]
您好，感谢反馈，我们会在接下来几个版本里支持服务端分页。

**Reply by owenlongbo** [reply, marked as answer]
已经支持
